trashcan.lua

```diff
@@ -25,10 +25,6 @@ minetest.register_node("pipeworks:trashcan", {
 				"item_image[0,0;1,1;pipeworks:trashcan]"..
 				"label[1,0;"..S("Trash Can").."]"..
 				"list[context;trash;3.5,1;1,1;]"..
-				default.gui_bg..
-				default.gui_bg_img..
-				default.gui_slots..
-				default.get_hotbar_bg(0,3) ..
```

Comment:
Should these be conditionally included?
I've not idea how much this changes outcome but if it's useful somewhere then could be easily handled similar way how `pipeworks.liquids.water.source` is done.

Reply:
no, because they return "" for legacy compat or are absolutely useless to the form